newsletter-template.md

```diff
@@ -1,6 +1,8 @@
----
-title: "This Month in Rust GameDev #{TODO} - {TODO} {TODO}"
----
++++
+title = "This Month in Rust GameDev #{TODO} - {TODO} {TODO}"
+date = 1970-01-01
```

Comment:
Hmmm. If I remember correctly, the date should be optional for drafts. Is it required here because of how the index template is written?

Reply:
Yeah, that plus the fact I'm using the presence of a date to distinguish between something that's conceptually a 'post' and something that's a static page... I think there's probably a better way of doing this though. Will tweak things.

Reply:
Yep, I was overcomplicating it - you can just set `page_template` in the section definition to override the template 😅 I've done that now, and made `date` optional for posts to render.